### PR TITLE
Thomas/add arabic language

### PR DESCRIPTION
**Pull request by balzdur**
In this PR :
- add missing arabic translations
- improve CSS on the whole app to handle `rtl` and `ltr`
- rename some components to be direction agnostic (naming follow the flexbox convention)

Bonnus: I removed a lot of `<ScrollArea />` usage in the app, something I want to deprecate so it's a win/win

### CODE REVIEW

**Comment by balzdur** [comment]
@morghim, if you have time, it would be nice to look at the added translations (only). They are all auto-generated.

In the future, I propose you open an issue with any "weird" UI / bad translation you find in the product, so we can iterate to ensure a godd user experience

**Review by Pascal-Delange — COMMENTED** [review]
ok,
To be really honest, I'm a bit conflicted about the the whole RightPanel=>EndPanel etc renaming.
It really sounds like too much of an abstraction for the typical developer who'll work on it and read the code (in other words: I don't think anyone will ever call the thing an "end panel" in natural language, so the component also shouldn't be named like that)
I understand "right" is no longer correct in the context of arab/hebrew, but I think it's a fair tradeoff that left/right is inverted in those contexts.

All good for the ltr/rtl flags however.

**Review by morghim — APPROVED** [review]
@balzdur i reviewed all added translations everything is good

**Comment by balzdur** [comment]
I reverted to `Right` / `Left` for component renaming + moved to `rtl:` `ltr:` selector to end/start logical property to reduce the CSS boilerplate (source [here](https://web.dev/learn/design/internationalization?hl=fr))

File: packages/app-builder/src/components/Decisions/DecisionEndPanel.tsx [hunk 1]
```diff
@@ -95,26 +94,19 @@ function DecisionRightPanelContent() {
...
+        </span>
+        <RightPanel.Close />
+      </RightPanel.Title>
+      <AddToCase />
     </RightPanel.Content>
   );
 }
 
-export const DecisionRightPanel = {
+export const DecisionRightPannel = {
```
Comment (Pascal-Delange) [inline packages/app-builder/src/components/Decisions/DecisionEndPanel.tsx:109]:
```suggestion
export const DecisionRightPanel = {
```

**Review by Pascal-Delange — APPROVED** [review]
thanks 🙏

File: packages/app-builder/src/components/Decisions/DecisionEndPanel.tsx (hunk 1 above)
Comment (balzdur) [reply packages/app-builder/src/components/Decisions/DecisionEndPanel.tsx:109]: I'll do it in another PR

**Pull request merged**